storage-node-v10-ha-ra-grs-v4/README.md

```diff
@@ -0,0 +1,82 @@
+# Build a highly available app in Node.js with RA-GRS storage
```

Comment:
Please merge v3.md and v4.md into a single file and notate any differences between the SDKs, which should be minimal.